src/wasm-linker.cpp

```diff
@@ -387,10 +389,30 @@ bool hasI64ResultOrParam(FunctionType* ft) {
   return false;
 }
 
+void Linker::makeDummyFunction() {
+  assert(!out.wasm.table.names.size());
```

Comment:
Oh, and let's make this `assert(out.wasm.table.names.empty())`, that's more readable.